catalog-info.yaml

```diff
@@ -17,8 +17,6 @@ metadata:
   annotations:
     github.com/project-slug: RoadieHQ/marketing-site
     backstage.io/techdocs-ref: dir:.
```

Comment:
I think you need project ids no?

Reply:
I thought it would fall back to the project-slug

Reply:
oh yeah. I forgot about that one.